lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -149,3 +157,58 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     visibility: hidden;
   }
 }
+
+.nx-global-sidebar__other-content {
```

Comment:
I think this class only work correctly when used in conjunction with `nx-scrollable`.  Is that intended or should we just build the scrolling CSS into this class itself.  If use with `.nx-scrollable` is intended, it should be documented.

Reply:
> I think this class only work correctly when used in conjunction with `nx-scrollable`. Is that intended or should we just build the scrolling CSS into this class itself. If use with `.nx-scrollable` is intended, it should be documented.

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/e677742ec7626551ce3a9c7fdde7534188919e7d